app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -723,5 +718,23 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         // PWA must have been used within last 30 days to be considered "recently used" for the
         // telemetry purposes.
         const val PWA_RECENTLY_USED_THRESHOLD = DateUtils.DAY_IN_MILLIS * 30L
+
+        /**
+         * set a delay until we locate visual completeness queue.
```

Comment:
nit: locate -> run

We're just running the queue after the delay. That being said, when this method is renamed, this is explaining implementation details that are subject to change: I think this line can be changed in favor of the high-level goal we're trying to accomplish.